cnv/cnv_users_guide/cnv-attaching-vm-multiple-networks.adoc

```diff
@@ -27,4 +27,8 @@ in the previous section.
 
 include::modules/cnv-vm-create-nic-web.adoc[leveloffset=+1]
 
-include::modules/cnv-networking-wizard-fields-web.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/cnv-networking-wizard-fields-web.adoc[leveloffset=+1]
+
+Install the optional xref:cnv-installing-qemu-guest-agent.adoc[QEMU guest agent] on the virtual machine so that the host can d
```

Comment:
Please fix "display."

We usually back the xrefs up to the top level.

Reply:
Well that's just embarrassing...

Reply:
no worries - it's the downfall of line splitting!